lib/trento.pm

```diff
@@ -63,12 +66,97 @@ use constant TRENTO_AZ_PREFIX => 'openqa-trento';
 # Azure does not support dash or underscore in ACR name
 use constant TRENTO_AZ_ACR_PREFIX => 'openqatrentoacr';
 use constant CYPRESS_IMAGE_TAG => 'goofy';
+use constant CYPRESS_IMAGE => 'docker.io/cypress/included';
 
 =head1 DESCRIPTION 
 
 Package with common methods and default or constant  values for Trento tests
 
 =head2 Methods
+=cut
+
+
+=head3 get_trento_deployment
+
+Get the set of scripts for the Trento deployment
+=cut
+
+sub get_trento_deployment {
+    my ($self, $work_dir) = @_;
+
+    enter_cmd "cd ${work_dir}";
+    my $gitlab_token = get_var(TRENTO_GITLAB_TOKEN => get_required_var('_SECRET_TRENTO_GITLAB_TOKEN'));
```

Comment:
What is the difference between `TRENTO_GITLAB_TOKEN` and `_SECRET_TRENTO_GITLAB_TOKEN` ?

Reply:
It was a way to let the user specify its own token, but as the repo is fixed, it makes no more sense. Could be removed.

Reply:
You can change it later.